docs/items/index.md

```diff
@@ -13,10 +13,15 @@ Basic items that need no special functionality (think sticks or sugar) do not ne
 |      Method        |                  Description                  |
 |:------------------:|:----------------------------------------------|
 | `requiredFeatures` | Sets the required `FeatureFlag`s needed to see this item in the `CreativeModeTab` it is added to. |
+| `food`         | Makes the item eatable and is used to store information about the Hunger and Effects. |
```

Comment:
This sentence could probably be reworded to explain the concept better. Probably something that mentions FoodData iirc the class correctly.